.travis.yml

```diff
@@ -130,7 +130,6 @@ script:
   - ./.ci/travis.sh run-command "$CMD"
   - ./.ci/travis.sh remove-custom-mvn
   - ./.ci/travis.sh remove-adoptium-jdk
-  - ./.ci/travis.sh git-diff
```

Comment:
Let's keep execution in Travis for now.
Execution in azure should be in addition.
So Travis step and some azure steps should fail bin the same time for same reason.

Reply:
Pay attention it runs after `ci/travis.sh run-command "$CMD"` where "mvn test" could be executed.

Reply:
Changes Revert

Reply:
@romani this change is still here, are you good with this?

Reply:
Hm..
 Ok let's restore it's execution in Travis.

Reply:
Changes Done.

Reply:
make this line as ` ./.ci/validation.sh git-diff`

Reply:
Changes Done.